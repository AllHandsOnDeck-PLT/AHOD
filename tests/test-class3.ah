Card card
main:
{
  card = Card("R5", true, 5) 
  do PRINT(card.type)
  do PRINT(card.faceup)
  do PRINT(card.value)
  card = Card("R5", false, 5)
  do PRINT(card.faceup)
}
