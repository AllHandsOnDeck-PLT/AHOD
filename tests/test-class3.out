R5
1
0